src/indexer.rs

```diff
@@ -44,14 +44,17 @@ pub enum TraverseError {
     #[error("Timed out waiting for finalized blocks.")]
     Elapsed(#[from] Elapsed),
     #[error("UnknownDataError occured: {0}")]
-    UnkownDataError(#[from] UnknownDataError),
+    UnknownDataError(#[from] UnknownDataError),
+    #[error("Other error occured: ${0}")]
```

Comment:
```suggestion
    #[error("Other error occurred: ${0}")]
```
I'm a little bit dubious that adding the text "Other error occurred" adds meaningful context.